plugins/org.eclipse.n4js.n4jsx.model/pom.xml

```diff
@@ -24,6 +24,13 @@ Contributors:
 		<relativePath>../pom.xml</relativePath>
 	</parent>
 
+	<properties>
+		<skip.mwe2>false</skip.mwe2>
+		<mwe2-file-location>/src/org/eclipse/n4js/model/GenerateN4JSModel.mwe2</mwe2-file-location>
```

Comment:
don't refer to mwe2 workflow from other bundle (create new workflow in this bundle)